_targets.R

```diff
@@ -1,63 +1,104 @@
+## Load packages for targets script
 library(targets)
 library(tarchetypes)
 library(tibble)
 suppressPackageStartupMessages(library(tidyverse))
 suppressPackageStartupMessages(library(dplyr))
 
+## Load packages needed by functions below
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse",
                             "dataRetrieval",
                             "urbnmapr",
                             "rnaturalearth",
                             "cowplot",
-                            "lubridate"))
+                            "lubridate",
+                            'leaflet',
+                            'leafpop',
+                            'htmlwidgets')
+               )
 
 # Load functions needed by targets below
 source("1_fetch/src/find_oldest_sites.R")
 source("1_fetch/src/get_site_data.R")
 source("3_visualize/src/map_sites.R")
 source("2_process/src/tally_site_obs.R")
+source("2_process/src/summarize_targets.R")
 source("3_visualize/src/plot_site_data.R")
+source("3_visualize/src/plot_data_coverage.R")
+source('3_visualize/src/map_timeseries.R')
 
-# Configuration
+## create log folder
+dir.create('3_visualize/log/', showWarnings = F)
+
+## Configuration - define global vars
 states <- c('WI','MN','MI','IL', 'IN', 'IA')
 parameter <- c('00060')
 
-# Targets
+## Structure static branching Targets with tar_map()
+
+mapped_by_state_targets <- tar_map(
+  # Use state as suffix in branch target naming
+  names = state_abb,
+  # Use unlist = F so that we can reference only the branch targets from
+  unlist = FALSE,
+  # Task names passed into tar_map() via arg. 'values='
+  values = tibble(state_abb = states) %>%
+    mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
+
+  # Target 1 - Subset to states in selected states list
```

Comment:
I like the per-target comments here.  If you left out the numbering, that might make things easier down the road if you add, remove, or rearrange targets. Just a note, no need to change this.

Reply:
I had a similar thought actually 😄 👍